Helpers.gs

```diff
@@ -1,3 +1,28 @@
+/**
+ * Takes an intended frequency in minutes and adjusts it to be the closest 
+ * acceptable value to use Google "everyMinutes" trigger setting (i.e. one of
+ * the following values: 1, 5, 10, 15, 30).
+ *
+ * @param {integer} The manually set frequency that the user intends to set
+ * @return {integer} The closest valid value to the intended frequency setting 
+ */
+function getValidTriggerFrequency(origFrequency) {
+  if (origFrequency == null) 
+    return 1; // if param is empty, just return 1, the lowest valid value.
```

Comment:
To reduce server loads and also prevent issues caused by starting a sync while a long-running one is already in progress, I would suggest returning 15 here. It is also currently the default for new installs and seems a sensible default for most use cases.
(We're also not logging this case right now, should we?)

Reply:
This makes sense to me. I added a new commit that makes this change (and adds a log).

Reply:
Great! Now only the comment isn't accurate anymore 🙈

Reply:
haha, yeah, I saw those and fixed it in a couple of subsequent commits. Can you check again?